wiki/Guides/Using_custom_hitsounds/de.md

```diff
@@ -1,49 +1,68 @@
----
-outdated_translation: true
----
+# Eigene Hitsounds benutzen
 
-# Eigene Hitsounds hinzufügen
+In diesem Tutorial wirst du lernen, wie du eigene [Hitsounds](/wiki/Beatmapping) in deinen [Beatmaps](/wiki/Beatmap) verwendest.
 
-Von dem Thread: *[Wie man einen eigenen Hitsound hinzufügt. (von neonat)](https://osu.ppy.sh/community/forums/posts/3215699)*
+## Eigene Hitsounds erhalten
 
-## Eigene Hitsounds
+Um in deinen Beatmaps eigene Hitsounds zu verwenden, musst du erst einmal welche haben! Die [Bibliothek für selbst erstellte Hitsounds](/wiki/Guides/Custom_Hitsound_Library) ist eine großartige Ressource, um Ton-Samples wie Becken, Trommeln, Glocken, Pfeifen und mehr zu finden. Alternativ kannst du, falls du nicht das findest, was du suchst, auch deine eigenen Samples erstellen!
 
-Als erstes werden einige Hitsounds benötigt.
+Hitsounds sollten in den Formaten `.wav` oder `.ogg` gespeichert sein, da diese keine Wiedervergabeverzögerung haben und sich korrekt wiederholen.
 
-- Man kann sie aus der [Spieler-Hitsound-Bibliothek](/wiki/Guides/Custom_Hitsound_Library) bekommen.
-- Oder man erstellt Hitsounds selber.
+## Eigene Hitsounds hinzufügen
 
-## Manuell die Hitsounds in den Song Ordner einfügen
+Wenn du deine gewünschten Tondateien gefunden hast, verschiebe sie in den Ordner der Beatmap, in der du sie benutzen willst. Falls du nicht weißt, wo du diesen Ordner findest, folge diesen Anweisungen:
 
-Der Song Ordner kann mit folgenen Schritten gefunden werden:
+1. Öffne osu!.
+2. Wähle die `Edit`-Option aus.
+3. Navigiere zu deiner Beatmap und öffne sie.
+4. Klicke auf `Datei` (die Option am linken Ende des Navigations-Menüs).
+5. Klicke auf `Songs-Ordner öffnen`.
+6. Füge deinen Dateien hier ein.
 
-1. Öffne osu!
-2. Klicke auf Edit.
-3. Navigiere zu *deine Beatmap hier* und öffne sie.
-4. Klicke oben auf Datein (Files) (Das Menu oben-links, es ist die weiß gefärbte Leiste oben).
-5. Öffne den Song Ordner.
-6. *und füge die Hitsounds ein*
+Falls du osu! auf macOS ausführst, musst du vielleicht eine leicht andere Vorgehensweise wählen:
 
-## Die Wahl welche Art von Ton der Hitsound sein soll
+1. Rechtsklicke auf das osu!-Anwendungsicon und wähle `Paketinhalt anzeigen`.
+2. Finde den Ordner deiner Beatmap in `drive_c -> osu! -> Songs`.
```

Comment:
```suggestion
2. Finde den Ordner deiner Beatmap in `drive_c -> osu! -> Songs` (Sortierung nach `Zuletzt modifiziert` kann hilfreich sein).
```
Again, not a macOS user, so I can't verify if the translation `Zuletzt modifiziert` is correct.